.github/labeler.yml

```
@@ -1,94 +1,123 @@
-"area/apps":
-  - "apps/**"
-
-"area/apps > hash*":
-  - "apps/hash-*/**"
-  - "libs/@local/hash-*/**"
-
-"area/apps > hash.ai":
-  - "apps/hashdotai/docs/**"
-  - "apps/hashdotai/glossary/**"
-
-"area/apps > hash-api":
-  - "apps/hash-api/**"
-
-"area/apps > hash.design":
-  - "apps/hashdotdesign/**"
-
-"area/apps > hash.dev":
-  - "apps/hashdotdev/**"
-
-"area/apps > hash-graph":
-  - "apps/hash-graph/**"
-
-"area/apps > hash-realtime":
-  - "apps/hash-realtime/**"
-
-"area/apps > hash-search-loader":
-  - "apps/hash-search-loader/**"
-
-"area/blocks":
-  - "blocks/**"
-
-"area/deps":
-  - "**/patches/**"
-  - "**/Cargo.lock"
-  - "**/yarn.lock"
-
-"area/dx":
-  - ".vscode/**"
-
-"area/infra":
-  - "infra/**"
-  - "**/*.md"
-  - "**/.changeset/config.json"
-  - "**/.github/**"
-  - "**/.*ignore"
-  - "**/.markdownlint*"
-
-"area/infra > docker":
-  - "infra/docker/**"
-
-"area/infra > terraform":
-  - "infra/terraform/**"
-
-"area/libs":
-  - "libs/**"
-
-"area/libs > antsi":
-  - "libs/antsi/**"
-
-"area/libs > deer":
-  - "libs/deer/**"
-
-"area/libs > error-stack":
-  - "libs/error-stack/**"
-
-"area/libs > sarif":
-  - "libs/sarif/**"
-
-"area/tests":
-  - "tests/**"
-  - "**/tests/**"
-  - "**/test/**"
-
-"area/tests > integration":
-  - "tests/hash-backend-integration/**"
-
-"area/tests > playwright":
-  - "tests/hash-playwright/**"
-
-"type/eng > backend":
-  - "apps/hash-graph/**"
-  - "apps/hash-api/**"
-  - "libs/@local/**"
-
-"type/eng > frontend":
-  - "apps/hash-frontend/**"
-  - "libs/@hashintel/**"
-
-"type/legal":
-  - "**/LICENSE**"
-  - "**/NOTICE**"
-  - "**/THIRD_PARTY_NOTICES**"
-  - "**/THIRD_PARTY_LICENSES**"
+area/apps:
+- changed-files:
+  - any-glob-to-any-file: apps/**
+
+area/apps > hash*:
+- changed-files:
+  - any-glob-to-any-file: apps/hash-*/**
+  - any-glob-to-any-file: libs/@local/hash-*/**
+
+area/apps > hash.ai:
+- changed-files:
+  - any-glob-to-any-file: apps/hashdotai/docs/**
+  - any-glob-to-any-file: apps/hashdotai/glossary/**
+
+area/apps > hash-api:
+- changed-files:
+  - any-glob-to-any-file: apps/hash-api/**
+
+area/apps > hash.design:
+- changed-files:
+  - any-glob-to-any-file: apps/hashdotdesign/**
+
+area/apps > hash.dev:
+- changed-files:
+  - any-glob-to-any-file: apps/hashdotdev/**
+
+area/apps > hash-graph:
+- changed-files:
+  - any-glob-to-any-file: apps/hash-graph/**
+
+area/apps > hash-realtime:
+- changed-files:
+  - any-glob-to-any-file: apps/hash-realtime/**
+
+area/apps > hash-search-loader:
+- changed-files:
+  - any-glob-to-any-file: apps/hash-search-loader/**
+
+area/blocks:
+- changed-files:
+  - any-glob-to-any-file: blocks/**
+
+area/deps:
+- changed-files:
+  - any-glob-to-any-file: patches/**
+  - any-glob-to-any-file: Cargo.lock
+  - any-glob-to-any-file: **/Cargo.lock
```

It's stupid but we cannot include it if it does not exist.
```suggestion
  - any-glob-to-any-file: Cargo.lock
```